controllers/disruption_controller.go

```diff
@@ -168,6 +168,10 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, fmt.Errorf("error handling chaos pods termination: %w", err)
 	}
 
+	if err := r.sendWarnings(instance); err != nil {
```

Comment:
```suggestion
	if err := r.sendWarningsOnDisruptionSpec(instance); err != nil {
```

Reply:
approved too early haha :oops:

Reply:
Honestly just silly that github can't manage to correct every use of the function when you suggested the name change. It knows where they all are!